machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -63,6 +63,7 @@ The status of the control plane machine set after installation depends on your c
 1. AWS clusters that are upgraded from version 4.11 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
 2. GCP and Azure clusters that are upgraded from version 4.12 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
 3. Nutanix and {rh-openstack} clusters that are upgraded from version 4.13 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
+4. Installing a cluster on VWware vSphere, includes a generated `ControlPlaneMachineSet` CR and activates the CR by default. This is a Technology Preview feature for VMware vSphere. See link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features - Scope of Support].
```

Comment:
This TP feature that is applied automatically after installing a cluster means that our customers are automatically opted in to a TP feature. but would they typically have to manually opt in to a TP feature given the level of support?

Reply:
The UPI vSphere docs include a link to [Creating a compute machine set on vSphere](https://docs.openshift.com/container-platform/4.14/machine_management/creating_machinesets/creating-machineset-vsphere.html#creating-machineset-vsphere_creating-machineset-vsphere](https://docs.openshift.com/container-platform/4.14/installing/installing_vsphere/installing-vsphere.html#installation-machine-requirements_installing-vsphere)

The Supported cloud providers section is embedded in the Machine management docs, but should I include a link in the vSphere UPI and IPI docs? Then agin, is the TP feature only for IPI? For UPI, a customer would need to generate the `ControlPlaneMachineSet` CR and then activate?